executor/src/witgen/data_structures/padded_bitvec.rs

```diff
@@ -70,11 +70,33 @@ impl PaddedBitVec {
     }
 
     pub fn get(&self, row: usize, col: u64) -> bool {
+        self.check_bounds(row, col);
+        let (word_index, bit_mask) = self.to_word_and_bit_mask(row, col);
+        let word = &self.data[word_index];
+        (word & bit_mask) != 0
+    }
+
+    pub fn set(&mut self, row: usize, col: u64, value: bool) {
+        self.check_bounds(row, col);
+        let (word_index, bit_mask) = self.to_word_and_bit_mask(row, col);
+        let word = &mut self.data[word_index];
+        if value {
+            *word |= bit_mask;
+        } else {
+            *word &= !bit_mask;
+        }
+    }
+
+    fn check_bounds(&self, row: usize, col: u64) {
```

Comment:
Any reason this is not jjust part of `to_word_and_bit_mask`?

Reply:
True, that's better!